docs/api-reference/direct-sql/joins.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: JOINs with Text Search
+---
+
+<Note>
+  See the [overview](/api-reference/direct-sql/overview#getting-started) for
+  creating the necessary example data
+</Note>
+
+To demonstrate joins with text search, we need more data. This table represents orders with a foreign key back to the
```

Comment:
I like the demonstration a lot. It would be useful to have a paragraph explaining the use case for JOINs w/ text search at a high-level before diving into an example.

Reply:
Roger that

Reply:
Done:

Combining text search with JOIN operations unlocks powerful querying capabilities in relational databases, even for
routine tasks like joining products with their orders. This integration allows you to perform sophisticated searches
across related tables, leveraging full-text search to query complex text fields such as product descriptions or metadata.

By doing so, you can efficiently answer nuanced business questions—like identifying how many "shoes" were sold, checking
if a specific customer purchased a "keyboard," or finding unsold items matching certain attributes—all within the familiar
framework of SQL.

By integrating text search with JOINs, you can create more expressive and flexible queries that tap into both structured
and unstructured data, enhancing your ability to analyze information effectively.

Reply:
I like this. Given that this is a key feature advantage over Elasticsearch, I wonder if we should highlight that directly. This is an open question, more than a suggestion, and I'd be curious to hear @rebasedming's thoughts as well.

Reply:
Sweet! I'll let @rebasedming do a final review as he has his ways of writing extremely clear and concise explanations :)